### ISSUE TITLE
CI: Coldcard patch applying incorrectly

### ISSUE THREAD

**Issue by instagibbs** [bitcoin-core/HWI#499]
```
+ git am ../../data/coldcard-multisig.patch
error: unix/frozen-modules/pyb.py: does not exist in index
hint: Use 'git am --show-current-patch' to see the failed patch
Applying: Use linux unix socket address format
```

f.e. https://github.com/bitcoin-core/HWI/pull/498/checks?check_run_id=2174878669

**Comment by achow101** [comment]
Was trying to fix this in #496 but ran into some other issues with their build system. Got side tracked and haven't been able to look into it further, although I have emailed the coldcard people about it.

**Comment by achow101** [comment]
This was fixed in one of the big CI fix PRs.
